docs/running-on-coclobas.md

```diff
@@ -20,12 +20,11 @@ of them.
 You'll need the GCloud command line utility, installed and initialized
 [from here](https://cloud.google.com/sdk/downloads#interactive).
 
-Optionally set your preferred zone and cluster sizes before you get started:
+Optionally set your preferred zone before you get started:
 
 ```shell
-# disco.sh defaults
+# disco.sh default
 export GCLOUD_ZONE="us-east1-c"
```

Comment:
that's right! In that case, changing the zone will take two exports. Updating the document to reflect that.